### ISSUE TITLE
[PHP] Long array and short array scope are not same

### ISSUE THREAD

**Issue by ptcong** [sublimehq/Packages#470]
I think long array and short array syntax should have same scope, but currently, it give different scopes

`$a = array(|);`
`embedding.php text.html.basic meta.embedded.block.php source.php meta.array.empty.php punctuation.definition.array.end.php`

`$a = array[|]` 
`embedding.php text.html.basic meta.embedded.block.php source.php punctuation.section.array.end.php`

```
$a = array(
    embedding.php text.html.basic meta.embedded.block.php source.php meta.array.php 
);
```

```
$a = [
    embedding.php text.html.basic meta.embedded.block.php source.php 
];
```

**Comment by wbond** [comment]
I believe this is fixed by 81606041f923dcae5caa37fce6aae3f98d46b2cc

**Comment by wbond** [comment]
Also 410027a7dc96e71bc81f94494b365927aaddaa9c
